<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HTTP_Request_Dialog</class>
 <widget class="QDialog" name="HTTP_Request_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HTTP Download File</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>168</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pb_download">
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_quit">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>361</width>
     <height>110</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="l_url">
          <property name="text">
           <string>URL:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="l_directory">
          <property name="text">
           <string>Directory:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="l_default_file">
          <property name="text">
           <string>Default file:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="le_url"/>
        </item>
        <item>
         <widget class="QLineEdit" name="le_directory"/>
        </item>
        <item>
         <widget class="QLineEdit" name="le_default_file"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_launch_file">
      <property name="text">
       <string>Launch file after downloaded</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
